### ISSUE TITLE
Remove deprecated "DefaultTask.newOutputFile()" function

### ISSUE THREAD

**Issue by TAGC** [JLLeitschuh/ktlint-gradle#208]
Using the latest plugin version (7.1.0), I get this warning when building:

```
$ gradle clean build --warning-mode all

> Task :annotations:ktlintMainSourceSetCheck
The DefaultTask.newOutputFile() method has been deprecated. This is scheduled to be removed in Gradle 6.0. Please use the ObjectFactory.fileProperty() method instead.
```

Likely caused by this:

https://github.com/JLLeitschuh/ktlint-gradle/blob/369b03bbca2e857fae271bb7a70f068fdf25ee8f/plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintCheckTask.kt#L70

**Comment by Tapchicoma** [comment]
Related PR: #192

**Comment by TAGC** [comment]
Ah cheers, I saw #60/#70 but missed that one.

**Comment by Tapchicoma** [comment]
PR was merged, should be available in the next released version.
